### ISSUE TITLE
Array Declaration Exception

### ISSUE THREAD

**Issue by mohidmeer** [lzybkr/PSReadLine#2926]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was -245.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
```

### Screenshot

Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was -242.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
mohid  Desktop  14:41  $fonts="1Row",
>> "3-D",
>> "3D Diagonal",
>> "3D-ASCII",
>> "3x5",
>> "4Max",
>> "5 Line Oblique",
>> "Acrobatic",
>> "Alligator",
>> "Alligator2",
>> "Alpha",
>> "Alphabet",
>> "AMC 3 Line",
>> "AMC 3 Liv1",
>> "AMC AAA01",
>> "AMC Neko",
>> "AMC Razor",
>> "AMC Razor2",
>> "AMC Slash",
>> "AMC Slider",
>> "AMC Thin",

### Environment data

```console
PS Version: 5.1.19041.1151
PS HostName: ConsoleHost (Windows Terminal)
PSReadLine Version: 2.0.0-beta2
PSReadLine EditMode: Windows
OS: 10.0.19041.1 (WinBuild.160101.0800)
BufferWidth: 170
BufferHeight: 44
```

### Steps to reproduce

I was trying to declare an array of round about 300 strings in it these strings are just font names i want to use this array in loop to genarate all the outputs for saving some time rather,
i was using figlet (npm package) and i want to output all the possible fonts by passing that array
$fonts=['all fonts','name in it']
foreach($ i in $fonts ){
figlet -f  $font[i] HELLO
}

### Expected behavior

System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.

### Actual behavior

System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.

**Comment by daxian-dbw** [comment]
This issue was already fixed (see #1306). Please upgrade to the [2.1.0 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.1.0) from PowerShell Gallery.
See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions. Please let us know if you run into the same issue with the latest version.

**Comment by mohidmeer** [comment]
> This issue was already fixed (see #1306). Please upgrade to the [2.1.0 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.1.0) from PowerShell Gallery. See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions. Please let us know if you run into the same issue with the latest version.

Thanks A Lot  It Resoled The Issue
